instanceidhandler/v1/instanceidhandler_test.go

```diff
@@ -202,3 +202,46 @@ func TestInstanceIDToDisplayName(t *testing.T) {
 		})
 	}
 }
+
+func TestInstanceID_GetSlug(t *testing.T) {
+	type fields struct {
+		apiVersion    string
+		namespace     string
+		kind          string
+		name          string
+		containerName string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    string
+		wantErr error
+	}{
+		{
+			name: "alpine deployment",
+			fields: fields{
+				apiVersion:    "apps/v1",
+				namespace:     "default",
+				kind:          "ReplicaSet",
+				name:          "alpine-deployment-6487cfc4c7",
+				containerName: "alpine-container",
+			},
+			want:    "default-replicaset-alpine-deployment-6487cfc4c7-02db-3363",
+			wantErr: nil,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			id := &InstanceID{
+				apiVersion:    tt.fields.apiVersion,
+				namespace:     tt.fields.namespace,
+				kind:          tt.fields.kind,
+				name:          tt.fields.name,
+				containerName: tt.fields.containerName,
+			}
+			got, err := id.GetSlug()
+			assert.Equalf(t, tt.want, got, "GetSlug()")
+			assert.ErrorIs(t, err, tt.wantErr)
+		})
+	}
```

Comment:
question: how is this different from the test exactly above?

Reply:
right... didn't notice it when I clicked on generate unittest for fuction

Reply:
lemme merge

Reply:
Before you do it, how this test case would be different from the existing ones?

To me it seems that it checks the exact same case where there is a valid input without anything special we should care about. So why add it at all?

Reply:
to show Eran that we reproduce his particular case

Reply:
plus we don't have any replicaset

Reply:
Nah, it’s okay. Thankfully, it’s a cheap and straightforward test.